packages/@aws-cdk/aws-msk/test/cluster.test.ts

```diff
@@ -0,0 +1,469 @@
+import { ResourcePart, SynthUtils } from '@aws-cdk/assert';
+import '@aws-cdk/assert/jest';
+
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as kms from '@aws-cdk/aws-kms';
+import * as logs from '@aws-cdk/aws-logs';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as core from '@aws-cdk/core';
+import * as msk from '../lib';
+
+/* eslint-disable quote-props */
+describe('MSK Cluster', () => {
+  let stack: core.Stack;
+  let vpc: ec2.IVpc;
+
+  beforeEach(() => {
+    stack = new core.Stack();
+    vpc = new ec2.Vpc(stack, 'Vpc');
+  });
+
+  describe('created with default properties', () => {
```

Comment:
All tests under this `describe` can be rolled into one since they just assert on different parts of the template, without actually configuring anything. Just a single test with a big assertion on all the properties would be better.